cmd/otelcorecol/main.go

```diff
@@ -0,0 +1,39 @@
+// Code generated by "go.opentelemetry.io/collector/cmd/builder". DO NOT EDIT.
+
+// Program otelcontribcol is an extension to the OpenTelemetry Collector
+// that includes additional components, some vendor-specific, contributed
+// from the wider community.
+package main
+
+import (
+	"log"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/service"
+)
+
+func main() {
+	factories, err := components()
+	if err != nil {
+		log.Fatalf("failed to build components: %v", err)
+	}
+
+	info := component.BuildInfo{
+		Command:  "otelcorecol",
+		Description: "Local OpenTelemetry Collector binary, testing only.",
+		Version:  "0.41.0-dev",
```

Comment:
Thoughts on how to keep this version in sync as the version gets updated?

Reply:
This is generated from the builder-config.yaml, not sure how to keep that up to date.